esmvalcore/dataset.py

```diff
@@ -663,9 +663,15 @@ def files(self) -> Sequence[File]:
     def files(self, value):
         self._files = value
 
-    def load(self) -> Cube:
+    def load(self, **kwargs) -> Cube:
```

Comment:
This method is intended to be an easy-to-use method that 'just' loads the data. It calls many preprocessor functions, not just `esmvalcore.preprocessor.load`. Therefore I foresee problems later on when someone decides they also need to pass on keyword arguments to other functions and then there may be a need to disentangle those. Rather than getting into that, it would be easier to use and maintain if the `Datset.load` method just understands how the tool has been configured in config-developer (it has access to that information through the `session` object), figures out what the correct parameters to `esmvalcore.preprocessor.load` are and passes those in.

Reply:
I had that in the beginning, but I thought it's cleaner that way since `callback` is also passed via `settings`. Now we will get two different ways of specifying kwargs for `load`. Anyway, I will change it back (`callback` will be removed in v2.10.0 anyway).

(As far as I understand, `session` does not include information about the config-developer file, only the config-user file. But it's still super easy to access that information).

Reply:
Done